src/Sulu/Bundle/HttpCacheBundle/EventSubscriber/InvalidationSubscriber.php

```diff
@@ -89,17 +82,15 @@ class InvalidationSubscriber implements EventSubscriberInterface
      * @param string $environment - kernel envionment, dev, prod, etc
      */
     public function __construct(
-        HandlerInvalidatePathInterface $pathHandler,
-        HandlerInvalidateStructureInterface $structureHandler,
+        ?CacheManager $cacheManager,
```

Comment:
Instead of making this nullable: Would it make sense to not load this subscriber at all, if caching is not activated? From what I can grasp here, it doesn't do anything, if this is null.